.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: main
+on: [pull_request, pull_request_review]
+
+jobs:
+  install:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu-latest", "macos-latest" ]
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v4
+      - name: create conda env
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-update-conda: true
+          python-version: 3.9
```

Comment:
@jmtsuji You might want to have it test multiple Python versions. That way, we can ensure compatibility by making sure it runs on newer versions.

Reply:
See: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#using-multiple-python-versions

Reply:
Replied below -- thanks for the feedback!